test/util.dart

```diff
@@ -118,12 +119,14 @@ class RefTestCaseCollection<T> {
     final calls = [];
     return RefTestCase(
       name: name,
+      // ignore: unnecessary_lambdas, avoid_types_on_closure_parameters
       ref: (T value) => calls.add(value),
       verifyRefWasUpdated: (actualValue) => expect(calls, [same(actualValue)], reason: _reasonMessage(name)),
       getCurrent: () => calls.single,
     );
   }
 
+  // ignore: avoid_shadowing_type_parameters
   RefTestCase createRefObjectCase<T>() {
```

Comment:
Sorry, don't know how I missed this one; this should also not be ignored
```suggestion
  RefTestCase createRefObjectCase() {
```